packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -468,6 +468,11 @@ export default async (containerEl, shellClient) => {
     .constant('UNIVERSE', 'DEDICATED')
     .run(
       /* @ngInject */ ($rootScope, $transitions) => {
+        $transitions.onStart({}, () => {
+          $('#currentAction').modal('hide');
```

Comment:
Is it possible to add this function in an utils class in order to avoid duplicating code through this application ?

Reply:
It's two differents monolith.